sklearn/metrics/classification.py

```diff
@@ -1067,7 +1067,7 @@ def fbeta_score(y_true, y_pred, beta, labels=None, pos_label=1,
 
     The `beta` parameter determines the weight of recall in the combined
     score. ``beta < 1`` lends more weight to precision, while ``beta > 1``
-    favors recall (``beta -> 0`` considers only precision, ``beta -> inf``
+    favors recall (``beta -> 0`` considers only precision, ``beta -> +inf``
```

Comment:
@rth Is this the docstring you mean?
if not please let me know what docstring should be updated.
Thanks!

Reply:
I mean the,
```
beta : float
        Weight of precision in harmonic mean.
```
but nevermind, this description is enough.